.textlintrc

```diff
@@ -38,36 +38,72 @@ rules:
   spelling:
     language: en
     skipPatterns:
-    - '/placeos/i'
+    # for anything in terminology, add only the corrected version
+    - '/[0-9]+/g'             # numbers 
+    - '/\b(?:[A-Z]){2,}\b/g'  # acronyms
+    - '/\b\S*\.\S{2,4}/g'     # filenames
+    - '/analytics/i'
+    - '/automations/i'
     - '/backoffice/i'
-    - '/[A-Z]{2,}/'      # acronyms
-    - '/[0-9]+/g'        # numbers 
+    - '/(re)?balance(r)?/i'
+    - '/integrat(ion|or)[s]?/i'
+    - '/microservice[s]?/i'
+    - '/placeos/i'
+    - '/preconfigure[ds]?/i'
+    - '/subnet/i'
     - '/tokeniz(er|ed|ing)?/i'
     - AOpen
+    - auth
+    - bootstrapper
     - chatbot
     - Chromebase
     - ChromeBox
     - ChromeOS
+    - CloudFormation
+    - config
+    - ElastiCache
+    - Elasticsearch
+    - etcd
     - Fargate
+    - frontend
+    - Frontends
+    - GSuite
+    - init
     - Kubernetes
+    - Meraki
     - Mimo
     - Moki
     - Multicast
+    - NGINX
     - Optica
     - pre-
+    - proxying
+    - RethinkDB
     - SaaS
     - webhook
+    
 
 
 
   terminology:
     defaultTerms: true
     terms:
+    - ['back\W?office', 'Backoffice']
+    - ['ETCD', 'etcd']
+    - ['front[- ]end', frontend]
+    - ['frontends', 'Frontends'] # the application, can conflict with "frontend" as a term?
     - ['place\W?os', 'PlaceOS']
     - ['place\W?tech(nology)?', 'Place Technology']
-    - ['back\W?office', 'Backoffice']
-    - "Kubernetes"
+    - ['place\W?analytics', PlaceAnalytics]
     - ['saas', SaaS]
+    - "CloudFormation"
+    - "ElastiCache"
+    - "Elasticsearch"
+    - "GSuite"
```

Comment:
```suggestion
    - ["gsuite", "Google Workspace"]
```
https://support.google.com/a/answer/10075155?hl=en

Reply:
Suggest "Google Workspace (G Suite)" for clarity - many still refer to it as G Suite.